src/sentry/killswitches.py

```diff
@@ -232,10 +232,10 @@ class KillswitchInfo:
 
 
 def validate_user_input(killswitch_name: str, option_value: Any) -> KillswitchConfig:
-    return normalize_value(killswitch_name, option_value, strict=True)
+    return normalize_config(killswitch_name, option_value, strict=True)
 
 
-def normalize_value(
+def normalize_config(
```

Comment:
Isn't this inconsistent now, since this is referred to as `value` in all other methods?